public/index.ts

```diff
@@ -7,18 +7,84 @@ import {
   View3d,
   Volume,
   VolumeMaker,
-  VolumeLoader,
   Light,
   AREA_LIGHT,
   RENDERMODE_PATHTRACE,
   RENDERMODE_RAYMARCH,
   SKY_LIGHT,
 } from "../src";
-import { State } from "./types";
+import { IVolumeLoader, LoadSpec } from "../src/loaders/IVolumeLoader";
+import { JsonImageInfoLoader } from "../src/loaders/JsonImageInfoLoader";
+import { OMEZarrLoader } from "../src/loaders/OmeZarrLoader";
+import { OpenCellLoader } from "../src/loaders/OpenCellLoader";
+import { TiffLoader } from "../src/loaders/TiffLoader";
```

Comment:
For external users src/index.ts will cut down on all this importing, but I wonder if this might be worth a src/loaders/index.ts

Reply:
Agreed - I'll try to change this to use src/index.ts for all the imports, as if the test app were really using the packaged volume-viewer.